### ISSUE TITLE
scale greater than 1

### ISSUE THREAD

**Issue by miraan** [nicklockwood/ReflectionView#7]
Hi, is it possible to set a scale greater than 1? It seems if I set a scale greater than 1, the reflection doesn't get any longer.

**Comment by nicklockwood** [comment]
No. The problem is that the reflectionScale doesn't actually scale the image, it controls the distance at which it fades to zero. So value greater than 1 just means the full image is shown
